### DISCUSSION TITLE
Overriding priceSelectionStrategy throwing error

### DISCUSSION THREAD

**Discussion by HarkiratShopism1** [medusajs/medusa#5416] [Q&A] [answered]
I am trying to override the price selection strategy from the below mentioned document. But I am encountering an error when I try to build.
Attached the snap shot of the error and the code throwing up error. 

https://docs.medusajs.com/modules/price-lists/backend/override-price-selection-strategy 
![Screenshot 2023-10-18 180736](https://github.com/medusajs/medusa/assets/132342163/6f1e34bc-c5b7-442e-9c35-0aca368bec26)
![Screenshot 2023-10-18 180719](https://github.com/medusajs/medusa/assets/132342163/133e6d75-640d-4bb9-8eae-47d1b7f97f2f)

**Comment by shahednasser** [comment, marked as answer]
Thank you for reporting this issue! We've updated the guide to remove the outdated details. Please check it out again and let us know if there are any other problems you face.

**Reply by HarkiratShopism1** [reply]
Thank you
